0.2/spec/index.html

```diff
@@ -394,10 +394,26 @@ <h2>Version Directories</h2>
             version directory names.
         </p>
         <p>
-            Version directories MUST have a sub-directory called <code>content</code> if there are files present, and
-            SHOULD NOT contain a <code>content</code> sub-directory otherwise. There MUST be no other files or
-            directories as children of a version directory, other than an <a href="#inventory">inventory file</a>,
-            an <a href="#inventory-digest">inventory digest</a>, and a <code>content</code> directory.
+            Version directories MUST contain a designated content sub-directory if the object contains files to be
+            preserved, and SHOULD NOT contain this sub-directory otherwise. The name of this designated sub-directory
+            MAY be defined in the <a href="#inventory">inventory file</a> using the key "contentDirectory" with the
+            value being the chosen sub-directory name as a string, relative to the version directory. The value of the
+            key "contentDirectory" MUST NOT contain any filesystem delimiter characters. If the key "contentDirectory"
```

Comment:
More \<code>contentDirectory\</code> x 2

Reply:
Do we need to be more specific about what a "filesystem delimeter character" is?

Reply:
Resolved the contentDirectory changes; I'm unsure what to do about the delimiter issue. Essentially trying to prevent someone from defining contentDir as 'my/content'. Maybe restrict the values to alphanumerics?

Reply:
We could instead just conform to DNS hostname conventions; 'Valid characters for hostnames are ASCII(7) letters from a to z, the digits from 0 to 9, and the hyphen (-).  A hostname may not start with a hyphen.' Which is a bit limiting but certainly avoids the file delimiter issue and any potential filecase issues.

Reply:
I'll admit to struggling with this one. A restriction such as DNS hostname conventions seems intuitively reasonable but also arbitrary. Why would it be wrong to use a word in some language and character set other than English and ASCII? We don't put constraints like this anywhere else.

Reply:
Hm. I think that would cause more confusion than it solves.

https://i❤.ws/emojidomain/💩 is a valid DNS entry...

Reply:
I think we should says simply

> The `contentDirectory` value MUST NOT contain the forward slash (`/`) path separator.

This is consistent with our description of [Inventory](https://ocfl.io/draft/spec/#inventory) where we define the special meaning as a delimiter.